src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -2435,6 +2435,7 @@ public IActionResult GetPublicAccess(int contentId)
                 .Select(_umbracoMapper.Map<MemberDisplay>)
                 .ToArray();
 
+            //TODO: change to role store
```

Comment:
Would this also need a change to the role store? Or remain with the service?

Reply:
I assume the controller would use the `RoleManager`, rather than the store directly

Reply:
Changed to role manager wording

Reply:
Question still stands from Umbraco logic POV, how far are we going with replacing services with Role Manager in this PR?